Description: Fix FTBFS on kfreebsd
 freebsd.c:287:11: error: 'struct kinfo_proc' has no member named 'kp_proc'
Forwarded: https://sourceforge.net/support/tracker.php?aid=3525356
Author: Vincent Cheng <Vincentc1208@gmail.com>
Last-Update: 05-09-2012

--- a/src/freebsd.c
+++ b/src/freebsd.c
@@ -283,11 +283,7 @@
 	pthread_mutex_lock(&kvm_proc_mutex);
 	p = kvm_getprocs(kd, KERN_PROC_ALL, 0, &n_processes);
 	for (i = 0; i < n_processes; i++) {
-#if (__FreeBSD__ < 5) && (__FreeBSD_kernel__ < 5)
-		if (p[i].kp_proc.p_stat == SRUN) {
-#else
 		if (p[i].ki_stat == SRUN) {
-#endif
 			cnt++;
 		}
 	}
